src/preferences/dialog/dlgprefinterface.cpp

```diff
@@ -64,12 +64,14 @@ DlgPrefInterface::DlgPrefInterface(QWidget * parent, MixxxMainWindow * mixxx,
     ComboBoxLocale->model()->sort(0); // Sort languages list
     ComboBoxLocale->insertItem(0, "System", ""); // System default locale - insert at the top
 
-    //
     // Skin configurations
-    //
-    QString warningString = "<img src=\":/images/preferences/ic_preferences_warning.png\") width=16 height=16 />"
-        + tr("The minimum size of the selected skin is bigger than your screen resolution.");
-    warningLabel->setText(warningString);
+    QIcon sizeWarningIcon(":/images/preferences/ic_preferences_warning.svg");
+    QString sizeWarningString =
+            "<img src=\":/images/preferences/ic_preferences_warning.svg\" "
+            "width=16 height=16/>  " +
+            tr("The minimum size of the selected skin is bigger than your "
+               "screen resolution.");
```

Comment:
Maybe we should explain the implications better?

```suggestion
            tr("The minimum recommended size of the selected skin is bigger than your "
               "screen resolution. Some UI elements might be cut off or hidden.");
```

Reply:
Maybe we can also mention what size the skin expects?

Something like "The recommended minimum size of the selected skin (1920x1080px) is bigger than your screen resolution. For best experience, use Internet Explorer 5."

Reply:
Hmm, while that would be nice, the user can't do anything about it, except getting a bigger screen.
The only skin affected is LateNight which requires 1280px (I thought we decided to increas that to 1366px some months ago?), and users using 1024px nowadays don't have high expectations I guess ;)

Reply:
> Hmm, while that would be nice, the user can't do anything about it, except getting a bigger screen.

Maybe they have misconfigured their screen resolution, but yeah, probably not worth adding it.